onnxmltools/convert/common/_apply_operation.py

```diff
@@ -420,6 +420,39 @@ def apply_parametric_softplus(scope, input_name, output_name, container, operato
         # g = a * f
         apply_mul(scope, [fName, aName], output_name, container)
 
+
+def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=None, alpha=[1.0], beta=[1.0]):
+
+    name = _create_name_or_use_existing_one(scope, 'ScaledTanh', operator_name)
+    if container.target_opset < 9:
+        if len(alpha) != 1 or len(beta) != 1:
+            raise ValueError('alpha and beta must be 1-element lists')
+        op_type = 'ScaledTanh'
```

Comment:
`op_type` is not necessary. We can just do `container.add_node('ScaledTanh', input_name, output_name, **attrs)

Reply:
Thanks for the suggestion! Fixed in update.